modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/RustServerCodegen.java

```diff
@@ -24,22 +24,40 @@
 import io.swagger.v3.oas.models.media.ArraySchema;
 import io.swagger.v3.oas.models.media.FileSchema;
 import io.swagger.v3.oas.models.media.Schema;
-import io.swagger.v3.oas.models.media.StringSchema;
 import io.swagger.v3.oas.models.media.XML;
-import io.swagger.v3.oas.models.parameters.Parameter;
 import io.swagger.v3.oas.models.parameters.RequestBody;
 import io.swagger.v3.oas.models.servers.Server;
 import org.apache.commons.lang3.StringUtils;
-import org.openapitools.codegen.*;
```

Comment:
Why have you expanded the * imports?

Reply:
I configured the project according to the google_checkstyle.xml file provided in the project. According to line 49 it is forbidden to use star imports:
```xml
        <module name="AvoidStarImport"/>
```

And since I touched it my IDE formatted the changed file the imports so that those a checkstyle conform.

Reply:
Well, it was the only thing I configured from checkstyle into my IDE. I have this option per default so I checked the checkstyle rules if I can leave that feature on.